siriuspy/siriuspy/pwrsupply/pructrl/prucparms.py

```diff
@@ -631,11 +631,23 @@ class PRUCParmsFAC_DCDC:
         CONST_PSBSMP.V_I_LOAD_MEAN,
         CONST_PSBSMP.V_I_LOAD1,
         CONST_PSBSMP.V_I_LOAD2,
-        CONST_PSBSMP.V_V_LOAD,
         CONST_PSBSMP.V_V_CAPBANK,
-        CONST_PSBSMP.V_TEMP_INDUCTORS,
-        CONST_PSBSMP.V_TEMP_IGBTS,
-        CONST_PSBSMP.V_DUTY_CYCLE,)
+        CONST_PSBSMP.V_DUTY_CYCLE,
+        CONST_PSBSMP.V_V_INPUT_IIB,
+        CONST_PSBSMP.V_I_INPUT_IIB,
+        CONST_PSBSMP.V_I_OUTPUT_IIB,
+        CONST_PSBSMP.V_TEMP_IGBT_1_IIB,
+        CONST_PSBSMP.V_TEMP_IGBT_2_IIB,
+        CONST_PSBSMP.V_TEMP_INDUCTOR_IIB,
+        CONST_PSBSMP.V_TEMP_HEATSINK_IIB,
+        CONST_PSBSMP.V_V_DRIVER_IIB,
+        CONST_PSBSMP.V_I_DRIVER_1_IIB,
+        CONST_PSBSMP.V_I_DRIVER_2_IIB,
+        CONST_PSBSMP.V_I_LEAKAGE_IIB,
+        CONST_PSBSMP.V_TEMP_BOARD_IIB,
+        CONST_PSBSMP.V_RH_IIB,
+        CONST_PSBSMP.V_IIB_INTERLOCKS,
+        CONST_PSBSMP.V_IIB_ALARMS,)
     groups[_const.G_SCAN] = (
         # --- common variables
         CONST_PSBSMP.V_PS_STATUS,
```

Comment:
@anacso17 , the common variables section needs updating as well

Reply:
done!